core/src/main/java/org/geysermc/geyser/network/GameProtocol.java

```diff
@@ -81,16 +78,6 @@ public static BedrockCodec getBedrockCodec(int protocolVersion) {
         return null;
     }
 
-    /* Bedrock convenience methods to gatekeep features and easily remove the check on version removal */
-
-    public static boolean isPre1_20(GeyserSession session) {
-        return session.getUpstream().getProtocolVersion() < Bedrock_v589.CODEC.getProtocolVersion();
-    }
-
-    public static boolean isPre1_20_10(GeyserSession session) {
```

Comment:
any chance we can keep this one specifically?
It'll be needed in the recipe unlocking PR too

Reply:
It dropped to zero usages when my latency/keepalive translator PR was merged, but I forgot to remove it

I'll re-add it considering you'll be using it